spring-boot-project/spring-boot-dependencies/build.gradle

```diff
@@ -1548,7 +1548,7 @@ bom {
 			releaseNotes("https://github.com/apache/maven-war-plugin/releases/tag/maven-war-plugin-{version}")
 		}
 	}
-	library("Micrometer", "1.15.0-SNAPSHOT") {
+	library("Micrometer", "1.15.0-RC1") {
```

Comment:
Sorry, this change isn't necessary since Boot is already on snapshots. When I started making the changes for the PR, this was still on M3.